### PR TITLE
Add Windows packaging to create an MSI installer

### PR DESCRIPTION
**Pull request by metajack**
---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [x] These changes do not require tests because they require manual testing

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/11967)

### CODE REVIEW

**Comment by metajack** [comment]
This isn't quite ready to merge yet. Trying to figure out why bhtml isn't loading.

**Comment by Coder206** [comment]
@metajack Should we add the ico for Windows Servo icon? (#11315)

**Comment by metajack** [comment]
@Coder206 it's already there in resources

**Comment by Coder206** [comment]
@metajack Wait, what? So why is there a need for #11315?

**Comment by Coder206** [comment]
@metajack I don't see an .ico file for Windows... I just see servo.png

**Comment by larsbergstrom** [comment]
At first blush, it looks good to me.

@Coder206 https://github.com/metajack/servo/blob/3fcf7316728cbbe117fe466107540f08b3b119cf/resources/Servo.ico is in the commit...

**Comment by larsbergstrom** [comment]
Oh, one more change needs to be made:
https://github.com/servo/servo/blob/master/etc/ci/upload_nightly.sh#L34-L35

Has to be changed to upload the .msi instead of the .tar.gz.

File: support/windows/Servo.wxs.mako [hunk 1]
```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Product Name="Servo Tech Demo"
```
Comment (aneeshusa) [inline support/windows/Servo.wxs.mako]: Change this to "Servo Nightly" perhaps?

**Comment by metajack** [comment]
This latest version appears to work, but requires a hardcoded path, has screwy font metrics, and is really slow.

cc @larsbergstrom

**Comment by larsbergstrom** [comment]
lgtm

Did you want to push the fonts.conf changes first, or just land this? I don't want to mess up your local working directory while you build :-)

**Comment by metajack** [comment]
I should have a minor update with new fonts.conf stuff in a few mins.

**Comment by metajack** [comment]
@larsbergstrom Ok, this should be working now except that it still isn't using the windows subsystem, so you get the extra console window. I think we can land this as is for people to try and fix the two remaining problems in followups.

**Pull request merged**